src/AccessibilityInsights.SharedUx/Controls/CustomControls/CheckableTreeViewItem.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Windows;
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    internal class CheckableTreeViewItem : TreeViewItem
+    {
+        protected override DependencyObject GetContainerForItemOverride()
```

Comment:
nit: Could we list the methods in this file and in CheckableTreeView.cs in the same order? At first glance, it was difficult to determine how the classes were different